ynput/ayon-core
# Resolve: avoiding clips without mediapoolitems

## Changelog Description
When encountering a scenario where resolve timelineitems are not connected to any media pool items, it is possible that these items could be either an *Adjustment Clip* or a *Fusion Composition*.


## Testing steps
1. add to your timeline  `Adjustment Clip` (from effect panel)
2. open Loader and try to load anything 
3. no error should happen and clip should be loaded to timeline
